x/vesting/keeper/integration_test.go

```diff
@@ -514,6 +521,349 @@ var _ = Describe("Clawback Vesting Accounts - claw back tokens", Ordered, func()
 	})
 })
 
+// Ensure the vesting account has access to unlocked tokens in EVM interactions
+var _ = Describe("Clawback Vesting Accounts - Unlocked EVM Tokens", Ordered, func() {
```

Comment:
Can you work this into the Describe at ln 46? This looks like a duplicate, slightly adjusted setup